src/rtextures.c

```diff
@@ -1630,6 +1630,241 @@ Image ImageTextEx(Font font, const char *text, float fontSize, float spacing, Co
+                    {
+                        image_value = (float)((pixel & 0b0000111100000000) >> 8)*(1.0f/15);
+                    }
+                    else if (selected_channel == 2)
+                    {
+                        image_value = (float)((pixel & 0b0000000011110000) >> 4)*(1.0f/15);
+                    }
+                    else if (selected_channel == 3)
+                    {
+                        image_value = (float)(pixel & 0b0000000000001111)*(1.0f/15);
+                    }
+                    image_alpha = (float)(pixel & 0b0000000000001111)*(1.0f/15);
+
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R8G8B8A8:
+                {
+                    image_value = (float)((unsigned char *)image.data)[k + selected_channel]/255.0f;
+                    image_alpha = (float)((unsigned char *)image.data)[k + 3]/255.0f;
+
+                    k += 4;
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R8G8B8:
+                {
+                    image_value = (float)((unsigned char *)image.data)[k + selected_channel]/255.0f;
+                    image_alpha = 1;
+
+                    k += 3;
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R32:
+                {
+                    image_value = ((float *)image.data)[k];
+                    image_alpha = 1;
+
+                    k += 1;
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R32G32B32:
+                {
+                    image_value = ((float *)image.data)[k + selected_channel];
+                    image_alpha = 1;
+
+                    k += 3;
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R32G32B32A32:
+                {
+                    image_value = ((float *)image.data)[k + selected_channel];
+                    image_alpha = ((float *)image.data)[k + 3];
+
+                    k += 4;
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R16:
+                {
+                    image_value = HalfToFloat(((unsigned short *)image.data)[k]);
+                    image_alpha = 1;
+
+                    k += 1;
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R16G16B16:
+                {
+                    image_value = HalfToFloat(((unsigned short *)image.data)[k+selected_channel]);
+                    image_alpha = 1;
+
+                    k += 3;
+                } break;
+                case PIXELFORMAT_UNCOMPRESSED_R16G16B16A16:
+                {
+                    image_value = HalfToFloat(((unsigned short *)image.data)[k + selected_channel]);
+                    image_alpha = HalfToFloat(((unsigned short *)image.data)[k + 3]);
+
+                    k += 4;
+                } break;
+                default: break;
+            }
+
+            // verify threshold
+            unsigned char result_value = 0;
```

Comment:
Please, could you review [raylib naming conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md)